README.md

```diff
@@ -52,15 +52,21 @@ rails db:{create,migrate}
 ```
 5. RAILS CREDENTIALS STEPS
 ```
-ADD STEPS HERE
+Run this command to open your credentials file:
+  EDITOR="code --wait" bin/rails credentials:edit
+Add your API key to the credentials file, formatted something like this:
+  google:
+    GOOGLE_CLIENT_ID: hg39874yt1vfh394uhi
+    GOOGLE_CLIENT_SECRET: 934786012394687
+
+You can then call on this API key anytime like this!:
+Rails.application.credentials.google[:google_client_id]
```

Comment:
Hmm, for some reason this isn't working for me. Is it supposed to be ran just from the terminal at any time?
It's giving me `zsh: no matches found: Rails.application.credentials.google[:google_client_id]`
I also tried it while having `EDITOR="code --wait" bin/rails credentials:edit` open, with the same error.
maybe we can just omit this part for now?